test/dlc_tests/container_tests/bin/benchmark/run_mxnet_training_performance_gpu

```diff
@@ -18,9 +18,15 @@ echo Fetch s3 bucket >&2
 aws s3 sync s3://aws-ml-platform-datasets/imagenet/pass-through/ /home/ubuntu/data
 echo Fetch finished >&2
 git clone https://github.com/rahul003/deep-learning-benchmark-mirror.git ${HOME_DIR}/artifacts/mxnet/deep-learning-benchmark-mirror  && cd ${HOME_DIR}/artifacts/mxnet/
+NUM_EPOCHS=20
+if [ "$DEV_MODE" == "1" ]
+then
+  NUM_EPOCHS=3
+  echo Running in Benchmark dev mode >&2
```

Comment:
just a quick question -- are these stdout redirect problematic like the other ones you fixed in functional tests?

Reply:
It should not. Have not encountered any issue to directly print message by-through stderr.